doc/admin.md

```diff
@@ -0,0 +1,193 @@
+
+## Introduction
+
+This guide defines the procedure to install the FIWARE Testbed Deploy.
+
+For general information, please refer to GitHub's [README](https://github.com/telefonicaid/fiware-testbed-deploy/blob/develop/README.md).
+
+## Build and install
+
+### Requirements
+- This scripts has been tested on a Debian 7 system, but any other recent Linux
+  distribution with the software described should work
+
+The following software must be installed (e.g. using apt-get on Debian and Ubuntu,
+or with yum in CentOS):
+
+- Python 2.7
+- pip
+- virtualenv
+
+### Installation
+
+The recommend installation method is using a virtualenv. Actually, the installation
+process is only about the python dependencies, because the scripts do not need
+installation.
+
+- 1.- Create a virtualenv 'deleteENV' invoking *virtualenv deleteENV*
+- 2.- Activate the virtualenv with *source deleteENV/bin/activate*
+- 3.- Install the requirements running *pip install -r requirements.txt
+   --allow-all-external*
+
+## Running FIWARE testbed deploy
+
+### Deploying one testbed
+To deploy just a testbed, it is required just execute one script. However some variables should be exported before.
+
+    export OS_AUTH_URL=<the authentication URL for the keystone in the Cloud>
+    export OS_USERNAME=<a user with an account in the Cloud>
+    export OS_TENANT_NAME=<a project name from the account in the Cloud>
+    export OS_PASSWORD=<the password for an account in the Cloud>
+    export OS_REGION_NAME=<the region name>
+    export OS_USER_DOMAIN_NAME=<OpenStack user domain name>
+    export OS_PROJECT_DOMAIN_NAME=<OpenStack project domain name>
+    export BOOKED_IP=<a booked IP in your Cloud infrastructure to deploy the VM>
+    export Region1=<the name for the region of the Cloud to be deployed>
+
+    fiwaretestbeddeploy/launch_vm.py
+
+The *launch_vm.py* ends in a few seconds, showing the floating IP. Then, it is
+ possible to connect to the ubuntu account of the server (using the
+SSH key at *~/.ssh/testbedskuld_key*).
+
+    ssh ubuntu@floating_ip
+
+Although the VM is active, the installation is still running
+inside the VM and needs a few minutes to complete. Usually the installation
+process lasts between 10 and 20 minutes. The job is finished after the file
+*config_vars* is copied into the */home/ubuntu* folder inside the virtual machine.
+
+When the installation is finished, the credential may be loaded with *. ~/config_vars*.
+The command *nova list* shows a testing VM that has been created during the installation
+inside the testbed (that is, it is a virtual machine running inside the testbed
+virtual machine). The floating IP 192.168.58.201 is associated to this
+VM (that is the second IP of the pool, the first was assigned to the router). It is
+possible to connect to the server following this steps:
+
+    eval $(ssh-agent -s)
+    ssh-add ~/.ssh/testkey
+    ip a add dev br-ex 192.168.58.1/24
+    ssh cirros@192.168.58.201
```

Comment:
Please take a review, because to start talking about Debian7, then you use a ubuntu user to make ssh, afterward you say that need a cirros username to make ssh to the machine.